pinball/ui/data_builder.py

```diff
@@ -395,11 +396,17 @@ def _workflow_data_from_instances_data(instances):
                 if instance.start_time > start_time:
                     start_time = instance.start_time
                     last_instance = instance.instance
-            elif end_time is not None and instance.end_time > end_time:
-                start_time = instance.start_time
-                end_time = instance.end_time
-                status = instance.status
-                last_instance = instance.instance
+            elif end_time is not None:
```

Comment:
instance.end_time == None means this instance is running

instance.end_time == sys.maxint this happens in some failure scenarios, if the instance is aborted/failed in some case, the end time will be set to sys.maxint. I encountered this when using pinball for other job framework. This bug blocks workflow from being scheduled to run again.